packages/scaffolding-cli/src/__tests__/domain/infraAks.bootstrap-config.json

```diff
@@ -0,0 +1,20 @@
+{
+  "projectName": "test-app-infra-only",
+  "projectType": "infra",
+  "platform": "aks",
+  "deployment": "azdevops",
+  "advancedConfig": true,
+  "createConfig": false,
```

Comment:
we don't need to name files in camelCase. but let's leave it then.

Reply:
AB#1977